package.json

```diff
@@ -77,7 +78,9 @@
   "copyright.owner": "IBM Corp.",
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "lint-staged",
+      "pre-push": "npm run build"
```

Comment:
This is even worse. 
- `npm run build` takes more than a minute to complete, that's way too long waiting just to be able to push my changes to github. 
- What if I have a work-in-progress branch that does not pass the build but I want the code to be backed up on GitHub and/or open a WIP pull request to discuss the early version? IIUC, this new hook will not allow me to do that.